### ISSUE TITLE
pdagent is now supported on FreeBSD

### ISSUE THREAD

**Issue by feld** [PagerDuty/pdagent#108]
Please list this on your README

http://www.freshports.org/net-mgmt/py-pdagent/

**Comment by bgdnlp** [comment]
Yes, please do. I was looking to getting it from source when I found this issue. Can we help? It's a simple command to install it, the wording could be something like:

FreeBSD
---------
The agent has been ported to FreeBSD. To install from packages:
```
sudo pkg install py27-pdagent
```
